istio-operator/istio-operator.spec

```diff
@@ -75,6 +75,8 @@ rm -rf $RPM_BUILD_ROOT
 mkdir -p $RPM_BUILD_ROOT%{_bindir}
 pushd OPERATOR/src/github.com/maistra/istio-operator/tmp/_output/bin/
 
+./istio-operator --version
```

Comment:
does this add value?

Reply:
Yes, especially when we get to doing a multi-arch build as it can change for the minimum supported version.  It helps to validate the build.